reference/configuration/framework.rst

```diff
@@ -465,45 +953,211 @@ would be ``/images/logo.png?version=5``.
     The latter option is useful if you would like older asset versions to remain
     accessible at their original URL.
 
-profiler
-~~~~~~~~
+hinclude_default_template
+.........................
 
-.. versionadded:: 2.2
-    The ``enabled`` option was introduced in Symfony 2.2. Previously, the profiler
-    could only be disabled by omitting the ``framework.profiler`` configuration
-    entirely.
+**type**: ``string`` **default**: ``null``
 
-.. _profiler.enabled:
+Sets the content shown during the loading of the fragment or when JavaScript is
+disabled. This can be either a template name or the content itself. It becomes
+the service container parameter named ``fragment.renderer.hinclude.global_template``.
 
-enabled
+.. seealso::
+
+    See :ref:`book-templating-hinclude` for more information about hinclude.
+
+.. _reference-templating-form:
+
+form
+....
+
+resources
+"""""""""
+
+**type**: ``string[]`` **default**: ``['FrameworkBundle:Form']``
+
+A list of all resources for form theming in PHP. If you have custom global form
```

Comment:
We need to be *very* clear in the first sentence here that if you are you using Twig templates, you need to look at the other setting. This is *only* if you are non-traditional and are using PHP instead of Twig. You mention it accurately here, but not strongly enough quite yet for me :). And we should have a link to the right spot.